learn/create-task/index.md

```diff
@@ -179,8 +179,8 @@ a random slugid for use as `taskId`.
 <pre data-plugin="interactive-example">
 let slugid = require('slugid');
 
-// Generate a new taskId
-let taskId = slugid.v4();
+// Generate a new taskId (see [npm docs](https://www.npmjs.com/package/slugid))
+let taskId = slugid.nice();
```

Comment:
No, here we should use `taskcluster.slugid()` that way we don't have to tell people to use nice...

Just tell them to use `taskcluster.slugid()`.